config/jobs/kubernetes-sigs/cluster-api-provider-azure/OWNERS

```diff
@@ -10,11 +10,11 @@ approvers:
 - karataliu
 - khenidak
 - nader-ziada
+- shysank
 reviewers:
 - cpanato
 - juan-lee
 - mboersma
-- shysank
```

Comment:
can you also please add @jackfrancis and @jsturtevant so this is up to date with our owners file?

Reply:
done